### DISCUSSION TITLE
How to use Pillow package in conda environment

### DISCUSSION THREAD

**Discussion by devudu** [python-pillow/Pillow#5758] [Q&A] [answered]
Hi,

After so much struggle for around a week, I finally built pillow package from the source in windows using Python3.10.0 + VS2019  and now I am able to use raqm feature. Now raqm is what feature I was mainly looking for because I have to render Indic languages (complex Unicode text).

Now the code is working in default python environment. However, I created a conda environment for writing my code. Now I am sure if I install pillow using `conda install -n myenv pillow`, its going to install default package without raqm feature. How can I use the package I have already build in conda environment? I see Pillow-8.4.0.dev0-py3.10-win-amd64.egg lying in my python\lib\site-packages and as well as in pillow-src\dist. Is there a way to use this file to use in my other conda environments?

**Comment by radarhere** [comment, marked as answer]
I think this question might be better asked over at https://github.com/conda-forge/pillow-feedstock, where there is knowledge about conda and about Pillow.

**Reply by radarhere** [reply]
https://github.com/conda-forge/pillow-feedstock/issues/109 may be relevant.

**Comment by aclark4life** [comment]
What is a pillow feedstock (serious question)? 🤷

**Reply by radarhere** [reply]
https://github.com/conda-forge/conda-build-feedstock#about-conda-forge
> conda-forge is a community-led conda channel of installable packages. In order to provide high-quality builds, the process has been automated into the conda-forge GitHub organization. The conda-forge organization contains one repository for each of the installable packages. Such a repository is known as a feedstock.

**Reply by aclark4life** [reply]
Oh I see ... thanks @radarhere 

> The conda-forge organization contains one repository for each of the installable packages. Such a repository is known as a feedstock.

So ... it's a forge ... and the feedstock is ... raw material to facilitate the forging. Wah-wah. Reminds me of "cheeseshop" :smile:
